### PR TITLE
Update README.txt

### PR DESCRIPTION
**Pull request by trentbraidner**
spelling fixes

### CODE REVIEW

File: v2.0/source/README.txt [hunk 1]
```diff
@@ -32,7 +32,7 @@ as follows:
      the skeltal BIOS source code and the SYSINIT and SYSIMES object
```
Comment (venimus) [inline v2.0/source/README.txt]: skeletal?

Comment (trentbraidner) [reply v2.0/source/README.txt]: yep, fixed

**Comment by RobStand** [comment]
Are these spelling errors original to the source? If so, they should be preserved.

**Comment by PixleProgrambetaxl** [comment]
+1

**Comment by bitcrazed** [comment]
Thanks for the suggested changes, but these are errors in the original MS-DOS 2.0 readme, and will be preserved as-is.

As we state on the repo's readme, we will not be accepting changes to the original sources (nor associated docs, samples, etc.)

**Pull request closed without merging**